sys-block/nbdkit/nbdkit-1.32.4.ebuild

```diff
@@ -0,0 +1,108 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+WANT_LIBTOOL=none
+inherit autotools
+
+DESCRIPTION="NBD server with stable plugin ABI and permissive license"
+HOMEPAGE="https://gitlab.com/nbdkit/nbdkit"
+SRC_URI="https://download.libguestfs.org/nbdkit/$(ver_cut 1-2)-stable/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+
+IUSE="curl ext2 nbd gnutls libguestfs libssh libvirt lzma selinux torrent zlib zstd"
+
+RDEPEND="
+	virtual/libiconv
+	selinux? ( sys-libs/libselinux )
+	gnutls? ( net-libs/gnutls )
+	curl? ( net-misc/curl )
+	libssh? ( net-libs/libssh:= )
+	libvirt? ( app-emulation/libvirt )
+	zlib? ( sys-libs/zlib )
+	nbd? ( sys-libs/libnbd )
+	lzma? ( app-arch/xz-utils )
+	zstd? ( app-arch/zstd )
+	libguestfs? ( app-emulation/libguestfs )
+	ext2? ( sys-fs/e2fsprogs )
+	torrent? ( net-libs/libtorrent-rasterbar )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-lang/perl
+"
+
+PATCHES=(
+	"${FILESDIR}/nbdkit-1.32.4-automagics.patch"
+)
+
+src_prepare() {
+	default
+
+	# This test fails for some reason.
+	echo "#!/bin/sh" > tests/test-file-extents.sh
```

Comment:
|| die on external commands which may fail, but let's use a heredoc instead of two echos?